docs/src/man/hosting.md

```diff
@@ -54,54 +53,34 @@ Then call the [`Travis.genkeys`](@ref) function as follows:
-
-    openssl aes-256-cbc -K $encrypted_<something>_key -iv $encrypted_<something>_iv -in .documenter.enc -out .documenter -d
-
-Pro Tip: You can add it automatically by running with --add.
-```
-
-**Ignore this**. Documenter will handle it for you.
-
-Once complete you will need to add the public key displayed in the REPL to your repository -- just follow the instructions displayed in the REPL:
+where `"MyPackage"` is the name of the package you would like to create deploy keys for. The
+output will look similar the text below:
 
 ```
-Make sure to add .documenter.enc to the git repository.
-
-Make sure not to add .documenter to the git repository.
-Commit all changes to your .travis.yml.
-WARNING: removing private key.
-
-Add the following public deploy key to '<username>/MyPackage.jl' with write access
-
-ssh-rsa [...] username@host.local
+INFO: add the public key below to https://github.com/USER/REPO/settings/keys
+      with read/write access:
 
-on the following page:
+[SSH PUBLIC KEY HERE]
 
-    https://github.com/<username>/MyPackage.jl/settings/keys
+INFO: add a secure environment variable named 'DOCUMENTER_KEY' to
+      https://travis-ci.org/USER/REPO/settings with value:
 
-Then commit the '.documenter.enc' file. Do not edit '.travis.yml'.
-
-WARNING: removing public key.
+[LONG BASE64 ENCODED PRIVATE KEY]
 ```
 
-where `<username>` is your GitHub username, or the name of the organization that hosts the package, and `[...]` is the actual public key that has been generated. Once you've added the public deploy key to GitHub, then close the REPL and commit the `docs/.documenter.enc` file that was generated by [`Travis.genkeys`](@ref) to the repository.
-
-If you don't get a `docs/.documenter.enc` file, one possible reason is
-an outdated version of `travis`. At the time of this writing, version 1.8.2
-was known to work.
-
-## Travis Environment Settings
+Follow the instructions that are printed out, namely:
 
-You should now go to your Travis settings page and check that two new keys have been added with names similar to the following
+ 1. Add the public ssh key to your settings page for the GitHub repository that you are
+    setting up by following the `.../settings/key` link provided. Click on **`Add deploy
+    key`**, enter the name **`documenter`** as the title, and copy the public key into the
+    **`Key`** field.  Note that you should include **no whitespace** when copying the key.
+    Check **`Allow write access`** to allow Documenter to commit the generated documentation
+    to the repo.
 
-- `encrypted_e6b49e69746a_key`
-- `encrypted_e6b49e69746a_iv`
+ 2. Next add the long private key to the Travis settings page using the provided link. In
+    the **`Environment Variables`** section add a key with the name `DOCUMENTER_KEY` and the
+    value that was printed out. **Do not** set the variable to be displayed in the build
+    log. Then click **`Add`**.
```

Comment:
Since it's a potential security hazard, I was thinking that maybe we should have a big red box here, just in case. Something like the following perhaps:

```
    !!! warning "Security warning"
        To reiterate: make sure that the "Display value in build log" option is **OFF** for
        the variable, so that it does not get printed when the tests run.
        This base64-encoded string contains the *unencrypted* private key that gives full
        write access to your repository, so it must be kept safe.
        Also, make sure that you never expose this variable in your tests, nor merge any code
        that does. You can read more about Travis environment variables in
        [Travis User Documentation](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings).
```